src/main/resources/schema/provider.definition.schema.v1.json

```diff
@@ -71,6 +71,19 @@
                 }
             }
         },
+        "replacementStrategy": {
+            "type": "array",
```

Comment:
description might be good to add as well as [default value](https://json-schema.org/understanding-json-schema/reference/generic.html) since we are defaulting to something.

Reply:
Good idea. Will add description and default value